tubular/scripts/retire_one_learner.py

```diff
@@ -125,7 +125,7 @@ def _get_learner_and_state_index_or_exit(config, username):
         learner = config['LMS'].get_learner_retirement_state(username)
         learner_state_index = _get_learner_state_index_or_exit(learner, config)
         return learner, learner_state_index
-    except HttpNotFoundError:
+    except HTTPError:
```

Comment:
This change is not looked fully consistent could you please clarify why it works in our case?

Reply:
`125 - learner = config['LMS'].get_learner_retirement_state(username)` -  executed with context manager that catches error such as 404
according FAIL - performed